<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="firstName">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="firstName_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lastName">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lastName_label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="zipCode">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="city">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="city_label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>City</string>
    </property>
   </widget>
   <widget class="QLabel" name="zipCode_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Zip Code</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Details</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="details">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>481</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="details_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Details</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>598</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
